gobblin-data-management/src/main/java/gobblin/data/management/copy/hive/HiveCopyEntityHelper.java

```diff
@@ -434,17 +463,22 @@ private Partition getTargetPartition(Partition originPartition, Path targetLocat
     private final Partition partition;
     private final Properties properties;
     private Optional<Partition> existingTargetPartition;
+    private final EventSubmitter eventSubmitter;
 
     public PartitionCopy(Partition partition, Properties properties) {
       this.partition = partition;
       this.properties = properties;
       this.existingTargetPartition = Optional.fromNullable(targetPartitions.get(this.partition.getValues()));
+      this.eventSubmitter = new EventSubmitter.Builder(dataset.getMetricContext(), "dataset.hive.copy").
```

Comment:
Second param to EventSubmitter.Builder is inconsistent with HiveEntityCopyHelper; there the second argument was "hive.dataset.copy, here it is "dataset.hive.copy"